src/handler.js

```diff
@@ -32,7 +31,7 @@ class Handler {
     this.pq = await consume({
       client: this.pulseClient,
       bindings: this.bindings,
-      queueName: this.queueName,
+      queueName: 'notifications',
```

Comment:
in tc-gh, we have this h in the config file (a string instead of env variable). I wonder if there's any difference in doing it one way or another?

Reply:
I think the only reason to include things in config.yml, but not as an env var, is to allow different behavior between testing and production.  I don't see a great advantage to different queue names in testing..